site/content/en/docs/manual/advanced/single-shape.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Single Shape '
+linkTitle: 'Single Shape Mode'
+weight: 8
+description: 'Guide to annotating tasks using Single Shape mode'
+---
+
+The CVAT Single Shape annotation mode accelerates the annotation process and enhances
+workflow efficiency for specific scenarios.
+
+By using this mode you can label shapes with a chosen annotation shape when an image
+contains only a single object. By eliminating the necessity to select tools from the sidebar
+and facilitating quicker navigation between images without
+the reliance on hotkeys, this feature makes the annotation process significantly faster.
+
+See:
+
+- [Single Shape mode annotation interface](#single-shape-mode-annotation-interface)
+- [Annotating in Single Shape mode](#annotating-in-single-shape-mode)
+- [Query parameters](#query-parameters)
+- [Video tutorial](#video-tutorial)
+
+## Single Shape mode annotation interface
+
+The interface of the **Single Shape** annotation mode varies based on the selected annotation
+type and the dataset type.
+
+The image below displays the complete interface, featuring all available fields;
```

Comment:
Not really all are shown. `Shape selector` is missing (when selected label has type `any`, user must specify the type implicitly)

Reply:
You resolved the conversation, but it is still relevant